gui/widgets/matRad_GammaWidget.m

```diff
@@ -0,0 +1,567 @@
+classdef matRad_GammaWidget < matRad_Widget
```

Comment:
indentiation is messed up in this file which compromises readability of the code.

Reply:
see above comment